runtime/doc/lsp.txt

```diff
@@ -1962,8 +1962,9 @@ rpc_response_error({code}, {message}, {data})
                                                          *vim.lsp.rpc.start()*
 start({cmd}, {cmd_args}, {dispatchers}, {extra_spawn_params})
     Starts an LSP server process and create an LSP RPC client object to
-    interact with it. Communication with the server is currently limited to
-    stdio.
+    interact with it. Communication with the spawned process happens via
```

Comment:
I'm not sure if it makes sense to mention this here or if it would be clearer to remove the sentence altogether

Reply:
I think this is fine, and it's good to cross-reference the TCP option as you've done here.